src/main/java/com/cinema/shop/service/UserService.java

```diff
@@ -7,4 +7,6 @@
     User add(User user);
 
     Optional<User> findByEmail(String email);
+
+    User getById(Long id);
```

Comment:
```suggestion
    User get(Long id);
```
again, uniform method names everywhere

Reply:
changed to getById everywhere